apps/site/src/lib/config.ts

```diff
@@ -17,5 +17,11 @@ export const isFork =
 export const shouldUseDummyEmailService =
   !process.env.BP_AWS_REGION && !process.env.VERCEL;
 
+// @todo: look into making feature flags more advanced
```

Comment:
What does this todo entail? Do we want these feature flags to have self-contained configuration and be toggleable outside the .env?

Reply:
I added this due to [this thread](https://hashintel.slack.com/archives/C02LG39FJAU/p1672930192711349) started by @nonparibus, which suggests we want a better solution for feature-flagging down the line (for split-testing, gating functionality for users of certain tiers, etc). This will likely require researching/adopting some existing feature-flagging service, so was deemed out of scope for work I've done as part of this PR.

I've created a task and added it to the `@todo` in https://github.com/blockprotocol/blockprotocol/pull/888/commits/f40d177d991d8ea351548538bc04daa7e7b8d13b to better capture this.